test/user-info.test.js

```diff
@@ -0,0 +1,165 @@
+'use strict'
+
+const { test } = require('tap')
+const Fastify = require('fastify')
+const fastifyCookie = require('fastify-cookie')
+const fastifySession = require('fastify-session')
+const fastifySecureSession = require('fastify-secure-session')
+const fastifyCsrf = require('../')
+
+const sodium = require('sodium-native')
+const key = Buffer.alloc(sodium.crypto_secretbox_KEYBYTES)
+sodium.randombytes_buf(key)
```

Comment:
Why not `crypto.randomBytes` so we don't need a native addon module?

Reply:
this is needed for fastify-secure-session.